pavelib/utils/test/utils.py

```diff
@@ -17,7 +17,7 @@ def clean_test_files():
     """
     Clean fixture files used by tests and .pyc files
     """
-    sh("git clean -fqdx test_root/logs test_root/data test_root/staticfiles test_root/uploads")
+    sh("git clean -fqdx test_root/logs test_root/data test_root/uploads")
```

Comment:
It depends on the type of tests right now... the paver unittest and javascript test tasks have a `skip_clean` command line option now that skips calling `clean_test_files` all together.  Perhaps we could build that out into the bok choy tasks too?